core/ecschema-editing/src/Merging/ClassMerger.ts

```diff
@@ -126,9 +126,9 @@ export class ClassMerger<TClass extends ECClass> {
   }
 
   // First pass to create missing changes
-  public static async mergeChanges(context: SchemaMergeContext, classChanges: Iterable<ClassChanges>) {
+  public static async mergeItemStubChanges(context: SchemaMergeContext, classChanges: Iterable<ClassChanges>) {
     const merger = new this(context);
-    const changes = Array.from(classChanges);
+    const changes = classChanges;
```

Comment:
there is no point in assigning this to a new variable now.  Using the input parameter directly makes the code clearer because the reader does not have to notice the assignment.  As they read the code they will see that the loop is over `classChanges` which is the input parameter.